sklearn/metrics/_dist_metrics.pyx.tp

```diff
@@ -853,17 +889,15 @@ cdef class BrayCurtisDistance(DistanceMetric):
 #------------------------------------------------------------
 # Jaccard Distance (boolean)
 #  D(x, y) = N_unequal(x, y) / N_nonzero(x, y)
-cdef class JaccardDistance(DistanceMetric):
+cdef class JaccardDistance{{name_suffix}}(DistanceMetric{{name_suffix}}):
     r"""Jaccard Distance
 
     Jaccard Distance is a dissimilarity measure for boolean-valued
     vectors. All nonzero entries will be treated as True, zero entries will
     be treated as False.
 
-    .. math::
-       D(x, y) = \frac{N_{TF} + N_{FT}}{N_{TT} + N_{TF} + N_{FT}}
```

Comment:
I removed such doc because this was making Tempita injection fail.

This can be reintroduced.

Reply:
Indeed it would be good to reintroduce it with a notation that does not use backslashes (assuming they are the ones causing the problem).